workflow-bot-app/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
     implementation platform('org.finos.symphony.bdk:symphony-bdk-bom:2.3.0')
 
     implementation 'org.finos.symphony.bdk:symphony-bdk-core-spring-boot-starter'
+    implementation 'org.finos.symphony.bdk:symphony-bdk-http-jersey2'
```

Comment:
As the workflow bot is relying on the Spring Boot Starter, shouldn't we use the `webclient` implementation instead of `jersey2`?